package com.demo.springconcepts.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import org.springframework.ui.Model;


@Controller
public class HelloController{

    @GetMapping("/hello")
    public String hello(Model model) {
        model.addAttribute("message","Hello from BridgeLabs");
        return "hello";
    }
}